cmd/chainbridge/deploy.go

```diff
@@ -0,0 +1,118 @@
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var deployCommand = cli.Command{
+	Action:   handleDeployCmd,
+	Name:     "deploy",
+	Usage:    "deploys test ganache",
+	Category: "TESTING",
+	Flags:    DeployFlags,
+	Description: "\tThe deploy command is used to deploy a ganache instance for testing\n" +
+		"\tTo add additional accounts, -testaccount\n" +
+		"\tTo only use custom accounts, -reset\n" +
+		"\tTo use a mnemonic, -mnemonic",
+}
+
+// The Private Keys of Alice, Bob, Etc
+var BaseAccounts = []string{
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.AliceKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.BobKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.CharlieKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.DaveKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.EveKey].Encode()),
+}
+
+// deployArgd holds the args to run tests with ganache
+type deployArgs struct {
+	port     string
+	amount   string
+	mnemonic string
+	accounts []string
+}
+
+func newdeployArgs() *deployArgs {
+	return &deployArgs{
+		accounts: BaseAccounts,
+		port:     "8545",
+		amount:   "100000000000000000000",
+		mnemonic: "",
+	}
+}
+
+func handleDeployCmd(ctx *cli.Context) error {
+	log.Info("Starting Ganache")
+
+	args := newdeployArgs()
+
+	args.port = ctx.String(PortFlag.Name)
+	args.amount = ctx.String(AmountFlag.Name)
+	args.mnemonic = ctx.String(MnemFlag.Name)
+
+	if newAccounts := ctx.String(AccountFlag.Name); newAccounts != "" {
+		accountList := strings.Split(newAccounts, ",")
+		if ctx.Bool(ResetFlag.Name) {
+			args.accounts = accountList
+		} else {
+			args.accounts = append(args.accounts, accountList...)
+		}
+	}
+
+	err := RunGanache(args)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Takes the Args struct and converts it to a string array to be passed into RunGanache
+func (a *deployArgs) ConvertToStringArray() []string {
+	args := []string{}
+
+	args = append(args, "-p", a.port)
+	for _, val := range a.accounts {
+		input := val + "," + a.amount
+		args = append(args, "--account", input)
+	}
+	if a.mnemonic != "" {
+		args = append(args, "-m", a.mnemonic)
+	}
+	return args
+}
+
+// RunGanache takes an input string and the gopath and run ganache-cli with the given inputs
+func RunGanache(args *deployArgs) error {
+
+	log.Info("Running npm install")
+	command := exec.Command("npm", "install")
+	command.Dir = "./on-chain/evm-contracts"
+	command.Stdout = os.Stdout
+	command.Stderr = os.Stderr
+	err := command.Run()
+	if err != nil {
+		return err
+	}
+
+	log.Info("Executing Ganache")
+	command = exec.Command("./node_modules/.bin/ganache-cli", args.ConvertToStringArray()...) //nolint:gosec
+	command.Dir = "./on-chain/evm-contracts"
+	command.Stdout = os.Stdout
+	command.Stderr = os.Stderr
+	err = command.Start()
```

Comment:
This doesn't really provide a useful interface. If we run this, ganache starts and will output to stdout, but the command returns so there is no way to kill it. 

For this command, I think using `Run()` would be sufficient (please try it out and see). When we want to use this in tests, I think `Start` is what we want but stdout/err should be directed elsewhere.

Reply:
The issue with is that it makes the go instance effectively hang until interrupted (eg if there is any code after `command.Run()` it will never be reached). If that is alright side effect, I can change it to `Run()`